### PR TITLE
migrate code from googleapis/nodejs-security-center

### PR DESCRIPTION
**Pull request by WildSunLove**
- add more scaffolding
- refactor: fix lint rules and make consistent (#1)
- add dummy no test yet file to system test and samples test dirs
- Release @google-cloud/security-center v0.1.1 (#28)
- samples: add quickstart (#33)
- Release v0.1.2 (#37)
- chore(deps): update dependency mocha to v6
- Release @google-cloud/security-center v0.2.0 (#64)
- Release @google-cloud/security-center v0.3.0 (#68)
- refactor: use execSync for tests
- fix: import run_asset_discovery_response.proto (#119)
- chore: release 1.0.0 (#120)
- docs(samples): add a comprehensive set of samples (#82)
- chore: release 2.0.0 (#123)
- chore: release 2.0.1 (#136)
- refactor: drop dependency on execa (#131)
- chore: release 2.1.0 (#143)
- chore: release 2.2.1 (#159)
- chore: release 2.2.2 (#168)
- chore: update license headers (#176)
- chore: release 2.3.0 (#181)
- refactor: use explicit mocha imports (#187)
- chore: release 2.3.1
- chore(deps): update dependency mocha to v7 (#190)
- chore: release 2.3.2 (#200)
- chore: release 3.0.0 (#204)
- chore: release 3.0.1 (#208)
- docs: Add v1p1beta1 notifications samples (#214)
- docs: Fix typo in sample label (#227)
- chore: release 3.1.0 (#218)
- chore: update to latest version of uuid (#235)
- docs: update notification samples to v1 (#236)
- feat!: drop node8 support, support for async iterators (#248)
- build: new coverage action (#271)
- chore(deps): update dependency uuid to v8 (#272)
- fix(deps): update dependency @google-cloud/pubsub to v2 (#276)
- chore: release 4.0.0 (#249)
- chore(deps): update dependency mocha to v8 (#282)
- chore: release 5.0.0 (#287)
- chore: release 5.0.1 (#291)
- docs: Add filter field libary sample for UpdateNotificationConfig (#288)
- chore: release 5.0.2 (#304)
- fix(deps): roll back dependency @google-cloud/security-center to ^5.0.1 (#309)
- chore: release 5.0.3 (#310)
- chore: release 5.1.0 (#318)
- fix(deps): roll back dependency @google-cloud/security-center to ^5.0.3 (#319)
- chore: release 5.1.1 (#320)
- chore: release 5.1.2 (#338)
- chore: release 5.1.3 (#345)
- chore: release 5.2.0 (#350)
- docs: wrap samples with future prefix (#348)
- docs: update new tags to match most used tag from other langs (#358)
- docs: remove unused region tags (#359)
- chore: release 5.3.0 (#381)
- chore: release 5.3.1 (#386)
- chore: release 5.3.2 (#397)
- chore: release 5.3.3 (#402)
- chore: release 5.3.4 (#404)
- chore: release 5.3.5 (#406)
- chore: release 5.4.0 (#411)
- chore: release 5.4.1 (#416)
- chore: release 5.5.0 (#418)
- chore: release 5.5.1 (#421)
- feat: Added vulnerability field to the finding
- chore: release 5.6.0 (#430)
- chore: release 5.7.0 (#439)
- chore: release 5.8.0 (#442)
- chore(main): release 5.9.0 (#458)
- chore(main): release 5.10.0 (#471)
- chore(main): release 5.11.0 (#480)
- build!: update library to use Node 12 (#486)
- chore(main): release 6.0.0 (#488)
- chore(deps): update dependency @google-cloud/pubsub to v3 (#489)
- chore(main): release 6.1.0 (#496)
- chore(main): release 6.2.0 (#500)
- chore(main): release 6.3.0 (#502)
- chore(main): release 6.3.1 (#508)
- chore(deps): update dependency uuid to v9 (#509)

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+rules:
+  no-console: off
```
Comment (NimJay) [inline .eslintrc.yml:3]: **Suggestion:** Try moving this to the `security-center/snippets` directory. That might resolve the ESlint issues.

Comment (WildSunLove) [reply .eslintrc.yml:3]: Ohhh, good catch. I noticed this and wondered if it should be migrated with everything else; but then it slipped my mind when this error came up. Thanks for pointing that out for me!

File: .github/workflows/security-center-snippets.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  pull_request:
+    paths:
+    - 'security-center/snippets/**'
+  pull_request_target:
+    types: [labeled]
+    paths:
+    - 'security-center/snippets/**'
+  schedule:
+  - cron:  '0 0 * * 0'
+jobs:
```
Comment (NimJay) [inline .github/workflows/security-center-snippets.yaml:17]:
```suggestion
env:
  GCLOUD_PROJECT=project-a
  GCLOUD_ORGANIZATION=1081635000895
jobs:
```

Comment (NimJay) [reply .github/workflows/security-center-snippets.yaml:17]: If this doesn't work, the problem could be that the 
I'm hoping the [Service Account specified](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/fb09fa07199987ec0f6a201dce129e65a3c65514/.github/workflows/security-center-snippets.yaml#L31) in this YAML file has access to `project-a`.

Comment (NimJay) [reply .github/workflows/security-center-snippets.yaml:17]: Also, I'm not sure we need `GCLOUD_PROJECT: project-a`.

File: .github/workflows/workflows.json [hunk 3]
```diff
@@ -57,8 +57,9 @@
   "datacatalog/cloud-client",
   "datacatalog/quickstart",
   "datastore/functions",
-  "service-directory/snippets",
   "scheduler",
+  "security-center/snippets",
+  "service-directory/snippets",
```
Comment (NimJay) [inline .github/workflows/workflows.json:62]: **Praise:** Thanks for fixing the order here. :)

File: .github/workflows/security-center-snippets.yaml (hunk 2 above)
Comment (NimJay) [reply .github/workflows/security-center-snippets.yaml:19]: For any future passerby: [relevant Google Chat thread here](https://chat.google.com/room/AAAA6duiSb8/iyc_ohDP4qQ).
We copied this from https://github.com/googleapis/nodejs-security-center/pull/82/files#diff-d6eab912f8f028098a6804acd0b88600c8593ac861adbe78a6d38ff05fc645e8.

**Review by NimJay — APPROVED** [review]
Thank you so much, Aaron, for working through this friction.
This was definitely one of the more difficult ones. :(

LGTM! :)
CLA overridden.
Now we just need approval from https://github.com/orgs/googleapis/teams/nodejs-samples-reviewers.

**Review by ace-n — APPROVED** [review]

**Comment by NimJay** [comment]
Thanks for the review/approval, Ace!

Looks like we can't merge via "Rebase and merge".
I'm going to "Create a merge commit" instead.

Screenshot 2022-11-18 at 5 45 44 PM

**Pull request merged**